### ISSUE TITLE
macOS Accessibility - wrong y position for elements out of the scroll view

### ISSUE THREAD

**Issue by dexterleng** [electron/electron#25155]
### Preflight Checklist

* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:**
  * What slack/discord/vscode is using 
* **Operating System:**
  * macOS Mojave 14.6 

### Expected Behavior

Typically in a normal macOS application using AppKit, when there is an `AXScrollArea`, the children elements when not visible in the scroll area have `AXPosition` of `nil`.

### Actual Behavior

However, in Electron, there is no `AXScrollArea`, there are no `AXScrollArea` inside the `AXWebArea`. `AXGroup` are used to represent the scrollable areas. When an element is positioned outside the scroll area, their y-coordinate ends at the edge of the AXGroup.

e.g. if a scrollable AXGroup has a frame of `x=0, y=0, width=100, height=100`, and an element out of the view would have a position of something like `x=50, y=100` or `x=50, y=0` (the x here is irrelevant) even when is completely non-visible.

### To Reproduce

I am developing an [application](https://vimacapp.com) ([repo](https://github.com/dexterleng/vimac)) that uses the accessibility API to show key bindings for (kinda like vimium for macos), and noticed a bunch of bindings at the bottom of the scroll area (see the screenshot), even though those elements are out of view. You can try it out on apps like slack and discord.

### Screenshots

<img width="950" alt="Screenshot 2020-08-27 at 6 42 03 PM" src="https://user-images.githubusercontent.com/34204380/91432991-21057b80-e895-11ea-875b-22a98c6caa76.png">

### Additional Information

I would love to help out!

**Comment by codebytere** [comment]
We require the template to be filled out on all new issues and pull requests. We do this so that we can be certain we have all the information we need to address your submission efficiently. This allows the maintainers to spend more time fixing bugs, implementing enhancements, and reviewing and merging pull requests.

We will be able to more closely look at your issue once you do so. This issue will be closed in 10 days if the above is not addressed.

**Comment by dexterleng** [comment]
@codebytere I have updated it.

**Comment by codebytere** [comment]
cc @felixrieseberg ☝🏻
